react-backend/client/src/components/visitors/signup_form.js

```diff
@@ -1,70 +1,90 @@
+  line-height: 1.5em;
+`;
+
+const CenteredHeading = Heading.extend`
+  width: 90%;
+  text-align: center;
+`;
+
+const TitleParagraph = Paragraph.extend`
+  font-weight: medium;
+  width: 90%;
+  font-size: 19px;
+  margin-top: -2px;
+  margin-bottom: -1em;
+`;
+
+const genders = [
+  { key: '1', value: '' },
+  { key: '2', value: 'male' },
+  { key: '3', value: 'female' },
+  { key: '4', value: 'prefer not to say' },
+];
 
 const signupForm = props => (
-  <div>
-    <form className="SignupForm" onChange={props.handleChange}>
-      <section className="Main col-9">
-        <h1>Please tell us about yourself</h1>
-        {props.error && (
-          <div className="ErrorText">
-            {props.error.map(el => <span key={el}>{el}</span>)}
-          </div>
-        )}
-        <Input name="fullname" question="Name" option="fullname" required />
-        <Input name="email" question="Email" option="email" required />
-        <Input name="phone" question="Phone Number (optional)" option="phone" />
-        <Select
-          name="gender"
-          question="Gender"
-          option="gender"
-          choices={['', 'male', 'female', 'prefer not to say']}
-          required
-        />
-        <Select name="fullname" question="Year of Birth" option="year" choices={props.years} required />
-        <button onClick={props.checkUserExists} className="Button" type="submit">
-          Submit
-        </button>
-      </section>
-      <aside className="Privacy col-3">
-        <h3 className="Privacy__title">
-          Why are we collecting this information?
-        </h3>
-        <p className="Privacy__text">
-          Here at XXXXXXXX, we take your privacy seriously: we will only use
-          your personal information to administer your account to provide the
-          products and services you have requested from us, and improve how we
-          deliver those.
+  <FlexContainerCol>
+    <CenteredHeading>Please tell us about yourself</CenteredHeading>
+    <Form className="SignupForm" onChange={props.handleChange}>
+      <FormSection flexOrder={1}>
+        <div>
+          {props.error && (
+            <div className="ErrorText">{props.error.map(el => <span key={el}>{el}</span>)}</div>
+          )}
+          <LabelledInput label="Full Name" name="fullname" option="fullname" required />
+          <LabelledInput label="Email Address" name="email" option="email" required />
+          <LabelledInput label="Phone Number (optional)" name="phone" option="phone" />
+          <LabelledSelect name="gender" label="Gender" options={genders} required />
+          <LabelledSelect name="year" label="Year of Birth" options={props.years} required />
+        </div>
+      </FormSection>
+      <FormSection flexOrder={2}>
+        <TitleParagraph>Why are we collecting this information?</TitleParagraph>
+        <CenteredParagraph>
+          Here at XXXXXXXX, we take your privacy seriously: we will only use your personal
```

Comment:
This is still `XXXXXXXXX`. This can't be in the version that's put out for user testing.

Reply:
Completely forgot about this, I'll fix

Reply:
This hasn't been changed.